sources/engine/Stride.BepuPhysics/.editorconfig

```diff
@@ -0,0 +1,190 @@
+# editorconfig.org
```

Comment:
As you know, the project started off as a repo outside of Stride and is now part of it. The style it started off on differs from stride's so I added this editor config in to ensure users contributing to those specific projects adhere to the existing style instead of Stride's.
This is likely fairly controversial, I can remove it if other contributors thinks this is a bad idea, but I would then have to refactor the code to adhere to Stride's style which will pollute the git history.